requirements/app/ui.txt

```diff
@@ -1,2 +1,3 @@
 streamlit >=1.13.0, <1.22.1
+altair <5.0.0  # strict - ModuleNotFoundError in streamlit: No module named 'altair.vegalite.v4'
```

Comment:
Isn't the correct fix to pin streamlit `# strict` until the fix their latest version?

Reply:
this was artifact and shall not be here